README.md

```diff
@@ -63,22 +63,125 @@ In this dialog, options should be filled out as follows:
+* FDASPROP Table Name: Generally this can be left blank. If you are trying to use the connector with a test instance of Data Access Studio, this should be FDASPROPT.
 
 The first time that you connect, you will also be asked for permissions to use to connect to the DataBox. This dialog looks something like this:
 
 ![](doc/DataBoxPermissions.png)
 
 You can either use Windows credentials for SQL Server or Basic (user name and password authentication) for either SQL Server or Oracle. In either case, you need to supply credentials that have
 the necessary permissions to access the DataBox database. Note that when using Data Access Studio and mobie, access to DataBox database data snapshots is controlled by the application. When using
-this Power BI connector, access must be granted directly to the database to use the connector. Click on OK and you will be presented with a complete list of snapshots available in the specified DataBox database.
-The names are listed in the form \<Report Name\>, \<Data Name\>, \<Snapshot Name\>:
+this Power BI connector, access must be granted directly to the database to use the connector.
+
+Beginning with release 1.0.0.3, it is recommended that you create specific database users for each of your mobie users that will be publishing and accessing data. The database user name
+should match one of the user identities associated with that mobie user in the mobie administration interface. For example, if you have a mobie user called TESTUSER and that user has a simple
+identity called TESTUSER associated with it, you would want to create a user called TESTUSER on your database. Windows user identities may also be used, particularly when using SQL Server.
+The TESTUSER database user should only be granted SELECT permission to the FDASPROP table in the mobie DataBox database. Beginning with build 8.0.11.1 of Data Access Studio, when TESTUSER
+publishes new data to the mobie DataBox, SELECT permission will also be granted to newly published data snapshots so that they are the only ones accessible to TESTUSER.
+
+In order to retain the behavior of the connector prior to release 1.0.0.3 that would show all available snapshots published by all users, it is necessary to set a value in a table called rn__Options
+in your mobie DataBox. See the release notes section below for details.
+
+Click on OK and you will be presented with a list of snapshots available in the specified DataBox database.
+The names are listed in the form \<Report Name\>, \<Data Name\>, \<Report Owner\>, \<Snapshot Name\>:
 
 ![](doc/Navigator.png)
 
 From here, you can select a snapshot and load it (or transform it) as you would any other Power BI data source.
 
+Beginning with release 1.0.0.3, the names displayed have changed to include the \<Report Owner\>. Typically, this will either be your mobie user name or the name DAS_PUBLIC to reference
```

Comment:
```suggestion
Beginning with release 1.0.3.0, the names displayed have changed to include the \<Report Owner\>. Typically, this will either be your mobie user name or the name DAS_PUBLIC to reference
```